.github/workflows/test-namui.yml

```diff
@@ -9,4 +9,4 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - run: docker build --target test ./namui
+      - run: docker build --target test -f namui/Dockerfile .
```

Comment:
Run docker from `namseent/` not `namseent/namui` to include `namui-cfg`.

Reply:
it's not a good idea because Dockerfile assume it run on the same directory. 
You need to find the other way to copy cfg file

Reply:
How about moving `namui-cfg/` crate down the `namui/` directory?
This is because other crates use re-exported `namui_cfg` through namui instead of calling namui-cfg directly.
In other words, we don't need to call Namui-cfg directly from other crates that use Namui

Reply:
discussion on slack https://projectluda.slack.com/archives/C02BNFY26NN/p1649925529501659?thread_ts=1649891582.830459&cid=C02BNFY26NN